alumet/tests/agent.rs

```diff
@@ -163,7 +163,7 @@ fn test_plugin_lifecycle() {
     let agent_config = AgentConfig::try_from(global_config).unwrap();
     let agent = agent.start(agent_config).unwrap();
     agent.pipeline.control_handle().shutdown();
-    agent.wait_for_shutdown().unwrap();
+    agent.wait_for_shutdown(Duration::from_secs(2)).unwrap();
```

Comment:
Did you tried other values ?

Reply:
Yes, but the test shouldn't take long. I've chosen 2 instead of 1 in case the test runs on a slow machine.